src/infra/team-maintenance.md

```diff
@@ -40,14 +70,12 @@ To make a full team member, the following places need to be modified:
 
 Remove the team member from any and all places:
 
-- 1password
-- The [GitHub team][gh-team], [GitHub nursery team][gh-nursery-team]
 - [team repo]
-- [toolstate notifications]
 - `triagebot.toml` files of all repos they were involved in
+- 1password
 
 [gh-team]: https://github.com/orgs/rust-lang/teams
 [gh-nursery-team]: https://github.com/orgs/rust-lang-nursery/teams
 [team repo]: https://github.com/rust-lang/team/tree/master/teams
 [team website]: https://www.rust-lang.org/governance
-[toolstate notifications]: https://github.com/rust-lang/rust/blob/master/src/tools/publish_toolstate.py
+[team-admins]: TODO
```

Comment:
Is the plan to open a PR against the team repo and then link once that's merged?

Reply:
Yes, that's my goal. I think we can discuss exact membership at that point - I don't have any strong feelings on that.